### PR TITLE
Load from file

### PR DESCRIPTION
**Pull request by jespicas**
I created a Test for LoadFromFile Button
To use cypress and upload files I added a dependency named cypress-file-upload
The test click on LoadFromFile
Upload a file and check if shown a text the file are upload.
Screenshot from 2020-10-03 15-47-07

### CODE REVIEW

File: cypress/integration/components/Tools/Buttons/EditorButton.spec.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
+/// <reference types="cypress" />
+
+context('Click Editor Button Go to Editor page and return to home page', () => {
```
Comment (aamay001) [inline cypress/integration/components/Tools/Buttons/EditorButton.spec.js:3]: Was this file included by accident? It should not be part of this PR. Can you please exclude this file from the PR?

**Review by aamay001 — CHANGES_REQUESTED** [review]

File: cypress/integration/components/Tools/Buttons/EditorButton.spec.js (hunk 1 above)
Comment (jespicas) [reply cypress/integration/components/Tools/Buttons/EditorButton.spec.js:3]: Yes! I will fix.

Comment (jespicas) [reply cypress/integration/components/Tools/Buttons/EditorButton.spec.js]: done

**Review by aamay001 — APPROVED** [review]
Thank you for your contributions!
